### ISSUE TITLE
Auto save

### ISSUE THREAD

**Issue by MKrabs** [zhrexl/ThisWeekInMyLife#13]
A really nice addition could be an auto-save feature, where the programm either saves after each action, periodically or on close.

**Comment by lunaposs** [comment]
I feel like it's probably bit redundant after we have made QoL saving changes, reducing the accidental loss of work quite significantly. Unless the system is significantly unstable in a manner that would have apps crash a lot, affecting Schedule even, I don't think there's going to be much demand for it, as autosaving is more ideal for intensive applications instead.

On a second though, we could probably implement a backup data instead, so that if you accidentally delete something and save without noticing and realize, you could replace new save with an older save and restore your previous work.

**Comment by zhrexl** [comment]
>On a second though, we could probably implement a backup data instead, so that if you accidentally delete something and save without noticing and realize, you could replace new save with an older save and restore your previous work.


I like that idea

**Comment by lunaposs** [comment]
Yeah! We could easily implement it, by just renaming the old file to have .old extension :)
